app/src/main/res/drawable/ic_icon_passthrough.xml

```diff
@@ -0,0 +1,28 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
Haven't checked the icon but where did you get it? What's the license? What's the common icon for that in the industry?

Reply:
I created it with Inkscape.